reducers/views/emoji.js

```diff
@@ -35,7 +35,7 @@ function shortcutReactToLastPostEmittedFrom(state = '', action) {
     }
 }
 
-function recentSkin(state = '', action) {
+function userSkinTone(state = '', action) {
```

Comment:
Did you mean to remove this?

Reply:
yes, I removed the old implementation but this one slipped through, thanks for catching it